<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main_Form</class>
 <widget class="QWidget" name="Main_Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {background-color: rgb(0, 0, 0); color: green}
QPushButton {background-color: rgb(168, 168, 168);}
QTextBrowser {background-color: rgb(236, 236, 236);}
QLineEdit {background-color: rgb(236, 236, 236); color: rgb(159, 80, 239)}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="login_label">
       <property name="font">
        <font>
         <family>System</family>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>ðŸ›¸</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="your_mes_label">
       <property name="font">
        <font>
         <family>System</family>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Your message:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="mes_lineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="send_all_pushButton">
       <property name="font">
        <font>
         <family>System</family>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Send to all</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="send_private_pushButton">
       <property name="font">
        <font>
         <family>System</family>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Send Private</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QTextBrowser" name="group_textBrowser">
       <property name="font">
        <font>
         <family>Bahnschrift</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="private_textBrowser">
       <property name="font">
        <font>
         <family>Bahnschrift</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="refresh_pushButton">
     <property name="font">
      <font>
       <family>System</family>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Refresh</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
